pnpm-workspace.yaml

```
@@ -65,13 +66,18 @@ catalogs:
     "expo-status-bar": ^2
     "expo-web-browser": ^14
 
+  next-stable:
+    next: ^15
+    "@next/bundle-analyzer": ^15
+    "@next/eslint-plugin-next": ^15
+    "@next/third-parties": ^15
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)